apps/docs/data/components/searchfield/SearchFieldH5.tsx.preview

```diff
@@ -0,0 +1,11 @@
+<SearchField

```

Comment:
Add example with box shadow (you can the search field at home page of h5 project)